packages/jest-worker/src/workers/ChildProcessWorker.ts

```diff
@@ -158,10 +158,20 @@ export default class ChildProcessWorker implements WorkerInterface {
   }
 
   getStdout(): NodeJS.ReadableStream {
+    if (this._child.stdout === null) {
+      throw new Error(
+        'stdout is null - this should never happen. Please open up an issue at https://github.com/facebook/jest',
```

Comment:
when actually this _can_ happen?

Reply:
I have no idea. It's from https://github.com/DefinitelyTyped/DefinitelyTyped/pull/33127

Reply:
felt better to add an explicit error rather than ignoring the type error